docs/technical_reference/property_models/ADM1.rst

```diff
@@ -139,13 +139,14 @@ Kinetic Parameters
    "First-order decay rate for X_pro, k_dec_X_pro", ":math:`k_{dec,X_{pro}}`", "k_dec_X_pro", 0.02, ":math:`\text{d}^{-1}`"
    "First-order decay rate for X_ac, k_dec_X_ac", ":math:`k_{dec,X_{ac}}`", "k_dec_X_ac", 0.02, ":math:`\text{d}^{-1}`"
    "First-order decay rate for X_h2, k_dec_X_h2", ":math:`k_{dec,X_{h2}}`", "k_dec_X_h2", 0.02, ":math:`\text{d}^{-1}`"
-   "Water dissociation constant, KW", ":math:`KW`", "KW", 2.08e-14, ":math:`(\text{kmol/}\text{m}^3)^2`"
+   "Water dissociation constant, pKW", ":math:`pKW`", "pKW", 14, ":math:`\text{dimensionless}`"
```

Comment:
```suggestion
   "Water dissociation constant, pKW", ":math:`pK_W`", "pKW", 14, ":math:`\text{dimensionless}`"
```

Reply:
I was debating whether we should explicitly describe in terms of "negative logarithm of ..." but I see below in constraints that that gets spelled out, so perhaps this is good enough.